app/presenters/groups_presenter.rb

```diff
@@ -8,6 +8,7 @@ def groups
       {
         name: list.name,
         contents: list.tagged_list_items.map(&:base_path),
```

Comment:
Should we flag this with a comment that it should be removed once migration to content_ids?